contracts/ZeroCollateral.sol

```diff
@@ -131,12 +131,64 @@ contract ZeroCollateralMain is ERC20Detailed, ERC20, MinterRole, Chainlink {
         decimals
     )
     {
+        require((daiAddress != address(0)) && (daoAddress != address(0)), "ZeroCollateral::constructor - invalid addresses");
+
         daiContract = daiAddress;
         zcDaoContract = daoAddress;
+
+        borrowCount = 0;
```

Comment:
All uint256 initialise as 0 on creation of the contract storage, so you only need the initialisation of `blockAccruedInterest`